pipelines/hf_radar/config/dataset.yaml

```diff
@@ -16,22 +16,22 @@ coords:
       standard_name: time
       units: Seconds since 1970-01-01 00:00:00 UTC
       timezone: UTC
-  lat:
-    dims: [lat]
+  latitude:
+    dims: [latitude]
     dtype: float32
     attrs:
       units: degree_N
       long_name: Latitude
       standard_name: latitude
-      _CoordinateAxisType: Lat
-  lon:
-    dims: [lon]
+      _CoordinateAxisType: latitude
+  longitude:
+    dims: [longitude]
     dtype: float32
     attrs:
       units: degree_E
       long_name: Longitude
       standard_name: longitude
-      _CoordinateAxisType: Lon
+      _CoordinateAxisType: longitude
```

Comment:
@jmcvey3 is the `_CoordinateAxisType` attribute from some set of standards/conventions? Is that a CF thing? Do you know what the values are supposed to be?

Reply:
Oh I probably copy-pasted this directly from the netCDF files stored on the SCCOOS database. Not sure it particularly matters if we include it or not.